mne/viz/_3d.py

```diff
@@ -473,36 +492,195 @@ def plot_trans(info, trans='auto', subject=None, subjects_dir=None,
+    elif isinstance(brain, str):
+        surfaces.append(brain)
+
     if meg_sensors is False:  # old behavior
-        meg_sensors = 'helmet'
+        meg = ['helmet']
     elif meg_sensors is True:
-        meg_sensors = ['helmet', 'sensors']
-    if eeg_sensors is False:
-        eeg_sensors = []
-    elif eeg_sensors is True:
-        eeg_sensors = 'original'
-    if isinstance(eeg_sensors, string_types):
-        eeg_sensors = [eeg_sensors]
-    if isinstance(meg_sensors, string_types):
-        meg_sensors = [meg_sensors]
-    for kind, var in zip(('eeg', 'meg'), (eeg_sensors, meg_sensors)):
+        meg = ['helmet', 'sensors']
+    elif isinstance(meg_sensors, str):
+        meg = [meg_sensors]
+    else:
+        meg = []
+    if ref_meg:
+        meg.append('ref')
+
+    return plot_alignment(info, trans=trans, subject=subject,
+                          subjects_dir=subjects_dir, surfaces=surfaces,
+                          coord_frame=coord_frame, meg=meg, eeg=eeg_sensors,
+                          dig=dig, ecog=ecog_sensors, src=src,
+                          mri_fiducials=mri_fiducials, bem=bem,
+                          verbose=verbose)
+
+
+@verbose
+def plot_alignment(info, trans='auto', subject=None, subjects_dir=None,
+                   surfaces=('head',), coord_frame='head',
+                   meg=('helmet', 'sensors'), eeg='original',
+                   dig=False, ecog=True, src=None, mri_fiducials=False,
+                   bem=None, verbose=None):
+    """Plot head, sensor, and source space alignment in 3D.
+
+    This function serves the purpose of checking the validity of the many
+    different steps of source reconstruction:
+
+    - Transform matrix (keywords ``trans``, ``meg`` and ``mri_fiducials``),
+    - BEM surfaces (keywords ``bem`` and ``surfaces``),
+    - sphere conductor model (keywords ``bem`` and ``surfaces``) and
+    - source space (keywords ``surfaces`` and ``src``).
+
+    Parameters
+    ----------
+    info : dict
+        The measurement info.
+    trans : str | 'auto' | dict | None
+        The full path to the head<->MRI transform ``*-trans.fif`` file
+        produced during coregistration. If trans is None, an identity matrix
+        is assumed.
+    subject : str | None
+        The subject name corresponding to FreeSurfer environment
+        variable SUBJECT. Can be omitted if ``src`` is provided.
+    subjects_dir : str
+        The path to the freesurfer subjects reconstructions.
+        It corresponds to Freesurfer environment variable SUBJECTS_DIR.
+    surfaces : str | list
+        Surfaces to plot. Supported values: 'head', 'outer_skin',
+        'outer_skull', 'inner_skull', 'brain', 'pial', 'white', 'inflated'.
+        Defaults to ('head',).
+        .. note: For single layer BEMs it is recommended to use 'brain'.
+    coord_frame : str
+        Coordinate frame to use, 'head', 'meg', or 'mri'.
+    meg : str | None | list
```

Comment:
It looks like the other channel types can be bool, so for consistency it would be nice to have it for MEG as well